sema/subyt/sources.py

```
@@ -241,7 +292,7 @@
             self._csvfile = open(self._csv, mode="r", encoding="utf-8-sig")
             return csv.DictReader(self._csvfile, delimiter=",")
 
-        def __exit__(self):
+        def __exit__(self, *exc):
```

Ruff (0.8.2)

Missing return type annotation for special method `__exit__`

(ANN204)